dispatcher.py

```diff
@@ -33,16 +31,10 @@ def resultSubscriber(uuid, sizeUrls, peers, htmls, addr):
         res = socket.recv_json()
         #nothing important to send
         socket.send(b"Done")
-        if res[0] != "RESULT":
-            continue
-        url, html = res[1:]
-        with open(f"results/html{i}", "w") as fd:
-            fd.write(html)
-        log.info(f"GET {url} OK")
-        i += 1
-        with lockResults:
-            htmls.append((url, html))
-            change = True
+        if res[0] == "RESULT":
+            log.info(f"GET {res[1]} {GREEN}OK{RESET}")
+            i += 1
+        msg_queue.put(res)
```

Comment:
This should reside into the if statement too?

Reply:
I decide that every message sended to a client will be divided in three parts, if you dont need to use some part, send `None` in it

Reply:
I understend your point. It is changed now.